components/credit-card/FinanceCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+
+export interface FinanceCardProps {
+    question: string,
+    answer: string,
+}
+
+const FinanceCard = ({question, answer}: FinanceCardProps) => {
+    const [color, setColor ] = useState("bg-gray-300")
+    const [borderColor, setBorderColor ] = useState("border-gray-500")
+    const [cardTitle, setcardTitle ] = useState("Click to Change")
+    
+    const onColorChangeRequested = () => {
+        if (color == "bg-gray-300") {
```

Comment:
you could add an OR operator to this condition and remove the last else if case.

ie `color === 'bg-gray-300' || color === 'bg-yellow-300'`